test/unit/module/real/core/is_infinite.cpp

```diff
@@ -0,0 +1,43 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/constant/valmin.hpp>
+#include <eve/constant/valmax.hpp>
+#include <eve/function/is_infinite.hpp>
+#include <eve/logical.hpp>
+#include <cmath>
+
+//==================================================================================================
+// Types tests
+//==================================================================================================
+EVE_TEST_TYPES( "Check return types of eve::is_infinite(simd)"
+              , eve::test::simd::all_types
+              )
+<typename T>(eve::as_<T>)
+{
+  using eve::logical;
+  using v_t = eve::element_type_t<T>;
+  TTS_EXPR_IS( eve::is_infinite(T())                    , logical<T>   );
+  TTS_EXPR_IS( eve::is_infinite(v_t())                  , logical<v_t> );
+};
+
+//==================================================================================================
+// Tests for eve::is_infinite
+//==================================================================================================
+
+EVE_TEST( "Check behavior of eve::is_infinite(simd)"
+        , eve::test::simd::ieee_reals
+        , eve::test::generate ( eve::test::ramp(0))
```

Comment:
this test looks like it doesn't test anything. We don't even test for eve::Inf/Minf.
We also test for all types YET we only test for IEEE reals there.

Reply:
changed